<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferredTranscriptsWidget</class>
 <widget class="QWidget" name="PreferredTranscriptsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Gene:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="f_gene">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="search_btn">
       <property name="text">
        <string>search</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>488</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="add_btn">
       <property name="toolTip">
        <string>Add preferred transcript</string>
       </property>
       <property name="icon">
        <iconset resource="GSvar.qrc">
         <normaloff>:/Icons/Add.png</normaloff>:/Icons/Add.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="DBTableWidget" name="table"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DBTableWidget</class>
   <extends>QTableWidget</extends>
   <header>DBTableWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="GSvar.qrc"/>
 </resources>
 <connections/>
</ui>
